content_manager/blocks.py

```diff
@@ -1,16 +1,23 @@
 from django import forms
+from django.forms.utils import ErrorList
 from django.utils.functional import cached_property
 from django.utils.translation import gettext_lazy as _, pgettext_lazy
 from dsfr.constants import COLOR_CHOICES, COLOR_CHOICES_ILLUSTRATION, COLOR_CHOICES_SYSTEM, IMAGE_RATIOS, VIDEO_RATIOS
 from wagtail import blocks
+from wagtail.admin.telepath import register
```

Comment:
J'ai une erreur chez moi, apparemment ça a bougé
```suggestion
from wagtail.telepath import register
```